pkg/config/config_test.go

```diff
@@ -20,10 +20,12 @@ func TestGetImagesFromJSONFile(t *testing.T) {
 		name: "Unmarshal images from file",
 		path: "images_file",
 		imagesContent: `{
-    "cloudControllerManagerAWS": "registry.ci.openshift.org/openshift:aws-cloud-controller-manager",
-    "cloudControllerManagerOpenStack": "registry.ci.openshift.org/openshift:openstack-cloud-controller-manager"
-}`,
+			"cloudControllerManagerOperator": "registry.ci.openshift.org/openshift:cluster-cloud-controller-manager-operator",
+    		"cloudControllerManagerAWS": "registry.ci.openshift.org/openshift:aws-cloud-controller-manager",
+    		"cloudControllerManagerOpenStack": "registry.ci.openshift.org/openshift:openstack-cloud-controller-manager"
+		}`,
```

Comment:
Why did the indentation change here? Also, it's inconsistent, please fix, here and below
```suggestion
    "cloudControllerManagerOperator": "registry.ci.openshift.org/openshift:cluster-cloud-controller-manager-operator",
    "cloudControllerManagerAWS": "registry.ci.openshift.org/openshift:aws-cloud-controller-manager",
    "cloudControllerManagerOpenStack": "registry.ci.openshift.org/openshift:openstack-cloud-controller-manager"
}`,
```
This could be a tabs vs spaces mix. I would expect these lines to be 1 tab indented and the closing brace to not be indented at all

I added suggestions for some of these but there are a lot of them